src/azure-cli-core/azure/cli/core/commands/client_factory.py

```diff
@@ -8,6 +8,7 @@
 import azure.cli.core._debug as _debug
 import azure.cli.core._logging as _logging
 from azure.cli.core.application import APPLICATION
+from azure.cli.core.telemetry._telemetry import log_event
```

Comment:
Shouldn't telemetry's __init__ import expose right stuff? Looks odd to import from private stuff like this...